### DISCUSSION TITLE
Please consider making nightly release

### DISCUSSION THREAD

**Discussion by mxvin** [rustdesk/rustdesk#1298] [General]
For fast paced project like this, I think we need to deliver nightly update for user that wants latest feature and improvement. 
Please, reconsider it. adding a bit of CI action won't hurt i guess?
Thanks

**Comment by mikekgr** [comment]
This is a needed action as far as I think. I have asked almost the same days ago but unfortunately  I did not got any answer or comment from the developers …
See:
https://github.com/rustdesk/rustdesk/discussions/1260

Best Regards.
